tests/sstable_3_x_test.cc

```diff
@@ -1732,7 +1732,13 @@ static void test_partition_key_with_values_of_different_types_read(const sstring
 SEASTAR_THREAD_TEST_CASE(test_uncompressed_partition_key_with_values_of_different_types_read) {
     auto abj = defer([] { await_background_jobs().get(); });
     test_partition_key_with_values_of_different_types_read(
-        UNCOMPRESSED_PARTITION_KEY_WITH_VALUES_OF_DIFFERENT_TYPES_PATH);
+        UNCOMPRESSED_PARTITION_KEY_WITH_VALUES_OF_DIFFERENT_TYPES_PATH, compression_parameters::no_compression());
+}
+
+SEASTAR_THREAD_TEST_CASE(test_compressed_partition_key_with_values_of_different_types_read) {
+    auto abj = defer([] { await_background_jobs().get(); });
+    test_partition_key_with_values_of_different_types_read(
+        COMPRESSED_PARTITION_KEY_WITH_VALUES_OF_DIFFERENT_TYPES_PATH, compressor::lz4);
```

Comment:
Should do a round with each available compressor.

Reply:
Done